### PR TITLE
Editan Theofilus Arifin

### PR DESCRIPTION
**Pull request by Theofilusarifin**
Menambahkan readme, approve dong plis 🙏🏻

### CODE REVIEW

**Comment by Theofilusarifin** [comment]
mas plis mas 😭😭😭

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # rakamin_1
 rakamin_1 adalah test repository
+
```
Comment (BenedictusAryo) [inline README.md:3]: parah banget sih masak kosong

**Review by BenedictusAryo — APPROVED** [review]
ok deh, approved

File: README.md (hunk 1 above)
Comment (Theofilusarifin) [reply README.md:3]: Makasih sepuh 🙏🏻😭

**Pull request merged**